thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlDetectionConfigTranslator.java

```diff
@@ -108,6 +108,5 @@ public DetectionConfigDTO generateDetectionConfig() {
    */
   protected void validateYAML(Map<String, Object> yamlConfig) {
     Preconditions.checkArgument(yamlConfig.containsKey(PROP_NAME), "Property missing " + PROP_NAME);
-    Preconditions.checkArgument(yamlConfig.containsKey(PROP_DESC_NAME), "Property missing " + PROP_DESC_NAME);
```

Comment:
I think description should be made compulsory to improve the overall experience of the alert recipients. We can discuss this later. Not urgent!